src/core/server/logging/README.mdx

```diff
@@ -1,3 +1,13 @@
+---
+id: kibCoreLogging
+slug: /kibana-dev-docs/services/logging
+title: Logging system
+image: https://source.unsplash.com/400x175/?Logging
```

Comment:
I assume the unsplash images are just a placeholder thing? I noticed these in a few other docs from the new system but don't see those images surfaced anywhere.

Reply:
They're a placeholder and are optional at this point. I added it simply to follow what I also saw was being done in other docs.